metricflow/test/model/validations/test_element_const.py

```diff
@@ -20,11 +20,11 @@ def test_inconsistent_elements() -> None:  # noqa:D
             UserConfiguredModel(
                 data_sources=[
                     DataSource(
-                        name="s1",
+                        reference="s1",
```

Comment:
Again, I think we want `name` here, not `reference`

Reply:
Updated.